src/components/NewLocationPage/SparkLineBlock/SparkLineBlock.style.tsx

```diff
@@ -0,0 +1,74 @@
+import { COLOR_MAP } from 'common/colors';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { mobileBreakpoint } from 'assets/theme/sizes';
+import { Chevron } from '../Shared/Shared.style';
+
+export const ChartTitleWrapper = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  margin-bottom: 0.5rem;
+
+  ${Chevron} {
+    transform: none;
+    margin-left: 0.25rem;
+  }
+`;
+
+export const ChartTitle = styled.span`
+  ${props => props.theme.fonts.regularBookMidweight};
+  font-size: 1rem;
+  max-width: 65px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const BlockTitle = styled.h2`
+  margin: 0 0 1.5rem;
+  font-size: 1rem;
+  line-height: 1.4;
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  text-transform: uppercase;
+`;
+
+export const SetContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+// Specific pixel dimensions via mocks
+export const SingleSparkLineContainer = styled.div`
+  width: 90px;
+  height: 48px;
+
+  @media (min-width: ${mobileBreakpoint}) {
+    width: 96px;
+  }
+
+  @media (min-width: 1000px) {
+    width: 120px;
+    height: 60px;
+  }
+
+  @media (min-width: 1320px) {
+    width: 96px;
+    height: 48px;
+  }
+`;
+
+export const StyledLink = styled(Link)`
+  color: black;
+  text-decoration: none;
+
+  &:not(:last-of-type) {
+    margin-right: 1rem;
+    margin-bottom: 1rem;
```

Comment:
The `margin-bottom` shouldn't be conditional right? Also, that's probably something we want to add to the container (`padding-bottom`). Consider moving contextual concerns to the container (like spacing between its children) instead of setting that on the children items themselves

```ts
const SparklineSetContainer = styled.div`
  padding-bottom: 1rem;
  * > {
    &:not(:last-of-type) {
      margin-right: 1rem;
    }
  }
`;

const StyledLink = styled(Link)`
  {/* only style what's inside the link */}
`;
```

```tsx
<SparklineSetContainer>
  <StyledLink to="/">
    <ChartTitle title={title} />
    <SparkLine rawData={rawData} smoothedData={smoothedData} />
  </StyledLink>
</SparklineSetContainer>
```

Reply:
im not sure i agree? `StyledLink` is the wrapper--i think it makes sense to put the margins on the children's wrappers, no? re: the margin bottom in particular, its a bit of a cheat way to make sure that there is spacing if the spark lines wrap

<img width="256" alt="Screen Shot 2021-04-21 at 2 23 37 PM" src="https://user-images.githubusercontent.com/44076375/115602668-2cd30180-a2ad-11eb-9a97-5237816b126a.png">